src/Classes/DbHandler.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace TopDog\Classes;
+
+/**
+ * Class DbHandler handles inputting breeds into db, inserting images into the db based on the breed, and retrieving the breed name, sub breed
```

Comment:
none of the other classes have comments - either delete this one or add to all others

Reply:
Pete

Reply:
Done